stepup/init-db.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
```

Comment:
Curiosity: why do you instruct the script to immediately exit upon error in this script, and not in the others? Is there a distinct advantage for doing it here?

TIL:  `set -x` is going to help me in the future while writing `bash`